zk_dtypes/tests/prime_field_factory_test.py

```
@@ -0,0 +1,854 @@
+
+  @parameterized.parameters("canonical", "mont")
+  def test_novel_prime_host_arithmetic(self, storage):
+    p = 10**9 + 7
+    dt = zk_dtypes.prime_field(p, storage)
+    av = [5, 999999999, p - 1, 123456, 0]
+    bv = [7, 999999999, 2, 654321, p - 1]
+    a = np.array(av, dtype=dt)
+    b = np.array(bv, dtype=dt)
+    self.assertEqual(
+        [int(x) for x in (a + b).tolist()],
+        [(x + y) % p for x, y in zip(av, bv)],
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)